### ISSUE TITLE
UI issue when editing context of MCQ subproblem

### ISSUE THREAD

**Issue by qdeconinck** [UCL-INGI/INGInious#702]
On inginious.org (INGInious v.0.8.dev5+gaa2e61c), the line numbers overwrite the text of the question.

![image](https://user-images.githubusercontent.com/5878547/131166049-5ec7a8dd-7390-4e16-a99f-b4f231f32557.png)

**Comment by Drumor** [comment]
Hello,

Thx for this issue, I can't saddly reproduce it for the moment.
Could u give me the way you produce this? With a description of your environment (browser,...)

Thx!

**Comment by qdeconinck** [comment]
Indeed, some information is missing :-)

The issue arises as a course administrator, when editing the task. The UI issue is only present when I just created a new problem (clicking on the blue `Add` button) and directly editing the context of this new problem. Interestingly, this issue is not present when editing the context of an already existing task. 

I just reproduced this bug on inginious.org using Brave Version 1.29.81 Chromium: 93.0.4577.82 (Official Build) (64-bit)
